### ISSUE TITLE
Pulling Activity Counts from Actigraph Data

### ISSUE THREAD

**Issue by miahoffmannd** [ghammad/pyActigraphy#93]
I was very excited to stumble across this API. However, I am using Actigraph (WGT3X) data to measure movement and need to pull the raw data of whether the individual is lying down, standing up, or sitting. This information is currently coded into the .agd file, but I cannot access it. Any help here would be great!

**Comment by ghammad** [comment]
Hello @miahoffmannd 

I just made some modification in the "feature/agd" branch in order to provide access to the positional information.

Could you confirm that the new methods:
- RawAGD.inclineOff
- RawAGD.inclineStanding
- RawAGD.inclineSitting
- RawAGD.inclineLying

provide access to the positional information you want?

I also implemented a new method, "inclinePosition" which returns a time series with the most probable state (maximum counts across all the "incline" states (Off, Lying, Sitting, Standing). It is also possible to map these states to integer (as done by default).
As usual, for more info:
```python
import pyActigraphy
help(pyActigraphy.io.agd.RawAGD.inclinePosition)
```

Once you confirm it is fine on your side with this new code, I'll merge into develop.

Hope that helps.

Greg

PS: if you like the package, please "star" it on GitHub and spread the words. Many thanks.

**Comment by miahoffmannd** [comment]
That worked great. Thank you!

**Comment by ghammad** [comment]
Hello @miahoffmannd 

Closed with #95 and thus included in the v1.1 release.

Happy actigraphing.
